resources/views/macros/form.twig

```diff
@@ -214,9 +214,10 @@ Renders a Bootstrap checkbox element with mb-3.
 @param {bool} [opt.disabled=false] - Whether to add the "disabled" attribute. Defaults to false.
 @param {bool} [opt.raw_label=false] - Whether to use the raw label value (=do not escape). Defaults to false.
 @param {string} [opt.info] - If set an additional info icon will be added to the label with the text as tooltip.
+@param {string} [opt.class="mb-3"] - CSS classes for the checkbox element. Defaults to "mb-3".
 #}
 {% macro checkbox(name, label, opt) %}
-    <div class="form-check mb-3">
+<div class="form-check {{ opt.class is defined ? opt.class : 'mb-3' }}">
```

Comment:
Spacing

Reply:
We need  a twig formatter / linter :see_no_evil:

Reply:
Done